lib/cocoapods-core/podfile/dsl.rb

```diff
@@ -368,6 +368,10 @@ def target(name, options = nil)
       # @option   options [Boolean] :show_env_vars_in_log
       #           whether this script phase should output the environment variables during execution.
       #
+      # @option   options [Symbol] :execution_placement
```

Comment:
Do you like `:execution_placement`? Would you prefer another word? I am not super happy but i cant find a better word.

Reply:
`position` ?

Reply:
I've thought about it and re-thought about it again. I like it. Let's go with this instead.